.github/workflows/podvm_smoketest.yaml

```diff
@@ -57,11 +59,11 @@ jobs:
 
       - name: Build image
         run: make image
-     
+
```

Comment:
This is an unrelated change, could you please remove it (in case we need to do a revert we want to keep this change separate)